From 897e6b0ee918c0f004f1efbb2c95522703014940 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 18 Dec 2023 15:43:55 +0100
Subject: [PATCH 0235/1695] dt-bindings: add RIF to default bindings config for
 stm32mp25

The RIF header is now part of default bindings header file for
stm32mp25.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 88d9b790f8f49bbede94b7620a86305311c539bf dt-bindings: add RIF to default bindings config for stm32mp25]
Change-Id: I3ee775467fff24694dcbccda4f4b4a867854c161
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index 4596df4e8..64c56a068 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -27,6 +27,7 @@
 #include <dt-bindings/clock/stm32mp25-clks.h>
 #include <dt-bindings/clock/stm32mp25-clksrc.h>
 #include <dt-bindings/reset/stm32mp25-resets.h>
+#include <dt-bindings/soc/stm32mp25-rif.h>
 #endif
 
 #endif /* __DRIVERS_STM32MP_DT_BINDINGS_H */
-- 
2.47.2

